patches/minecraft/net/minecraft/client/renderer/RenderGlobal.java.patch

```diff
@@ -93,29 +93,119 @@
                  {
                      this.field_175009_l.add(renderchunk4);
                  }
-@@ -1253,6 +1268,13 @@
+@@ -1253,13 +1268,15 @@
  
      public void func_174976_a(float p_174976_1_, int p_174976_2_)
      {
-+        net.minecraftforge.client.IRenderHandler renderer = this.field_72769_h.field_73011_w.getSkyRenderer();
-+        if (renderer != null)
-+        {
-+            renderer.render(p_174976_1_, field_72769_h, field_72777_q);
-+            return;
-+        }
-+
++        int renderPass = net.minecraftforge.fml.client.FMLClientHandler.instance().renderSky(p_174976_1_);
++        if(renderPass == -1) return;
          if (this.field_72777_q.field_71441_e.field_73011_w.func_186058_p().func_186068_a() == 1)
          {
-             this.func_180448_r();
-@@ -1470,6 +1492,7 @@
+-            this.func_180448_r();
++            if(renderPass == 0 || renderPass == 1) this.func_180448_r();
```

Comment:
You should put `if (renderPass == 0 || renderPass == 1)` on a separate line above `this.func_180448_r();` to keep this patch simple.